tests/components/roku/test_media_player.py

```diff
@@ -105,6 +110,7 @@ async def test_tv_setup(
 
     assert hass.states.get(TV_ENTITY_ID)
     assert tv
+    assert tv.device_class == DEVICE_CLASS_TV
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)